### DISCUSSION TITLE
how to doCLSM-HyMAP2 2-way coupling

### DISCUSSION THREAD

**Discussion by PriDeHydro** [NASA-LIS/LISF#1354] [Q&A] [answered]
Hi.. 
I want to do a 2-way coupling between CLSM and HyMAP2.
what are the changes required or new stuff to be included in comparison to 1-way coupling?

**Comment by agetiran** [comment, marked as answer]
Hi @PriDeHydro,
The 2-way coupling between CLSM and HyMAP hasn't been implemented and changes in the CLSM code would be needed in that implementation. HyMAP is currently 2-way coupled with NoahMP3.6. Details can be found in https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2021MS002463

I can support on the effort to 2-way couple CLSM and HyMAP, in case you are interested in implementing that capability

**Reply by PriDeHydro** [reply]
thanks @agetiran.. I have been going through this paper. It would be great to work on the 2-way coupling between CLSM and HyMAP.. however, for my current study, I just need to get the 2-way coupling running between LSM and HyMAP. I will try to set up the 2-way coupling between NoahMP3.6 and HyMAP.

{Please see the attached image] Are these the only two changes that needed to be done on the HyMAP part in the LIS config file for 2-way coupling between NoahMP 3.6 and Hymap? Or some thing more is needed to be considered or changed?
![2-way](https://github.com/NASA-LIS/LISF/assets/136304021/eb7fd027-d883-45c2-93bc-b08629db543a)

**Comment by agetiran** [comment]
Hi @PriDeHydro 
Yes, these are the two options needed to run the 2-way coupling. 

the first option should be set as 1, and the second one as any value below 1 (I suggest 0.1-0.2, but you can do a sensitivity analysis to optimize the results). Note that this capability hasn't been comprehensively tested, so you may find issues in your runs. I'll be glad to help with it, as I'm interested in these results. 
Also keep in mind that there’s still a bug that causes continuous groundwater depletion, leading to negative GW values in random grid points. It's an issue on the NoahMP side that I haven't been able to resolve yet.

**Reply by PriDeHydro** [reply]
Thanks @agetiran for the help. 
I will connect with you if any issue comes about..
